packages/upgradeable/src/DeploymentFactory.ts

```diff
@@ -1,55 +1,51 @@
-import { DeployFunction } from "@eonian/hardhat-deploy/types";
-import { BaseDeploymentConfig, BaseDeploymentService } from "./BaseDeployment.service";
-import { HardhatRuntimeEnvironment } from "hardhat/types";
+/// <reference path="types.d.ts"/>
+import type { DeployFunction } from 'hardhat-deploy/types'
+import type { HardhatRuntimeEnvironment } from 'hardhat/types'
+import type { BaseDeploymentConfig, BaseDeploymentService } from './BaseDeployment.service'
 
-export type Newable<T> = { new (...args: any[]): T; };
+export interface Newable<T> { new (...args: any[]): T }
```

Comment:
Why this change to type?

Reply:
hm, dunno, seems like it was fixed automatically